src/styles/ParentStyles/index.less

```diff
@@ -142,8 +143,79 @@
   margin: 5%;
 }
 
+//****Parent-Progress styles****
```

Comment:
This styling is very clearly organized, but could this style sheet take advantage of less functionality like nested selectors and variables to avoid repetition?

Reply:
That's a good point!